refinery/tool_manager/tasks.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+
+from celery.task import task
+from django_docker_engine.docker_utils import DockerClientWrapper
+
+
+@task()
+def docker_garbage_collection():
```

Comment:
Maybe rename this per Ilya's suggestion, but looks good to me otherwise

Reply:
renamed.

Reply:
@mccalluc  Renamed to what?